pkg/gatherers/clusterconfig/operators_pods_and_events.go

```diff
@@ -40,27 +40,43 @@ type CompactedEventList struct {
 	Items []CompactedEvent `json:"items"`
 }
 
-// Used to detect the possible stack trace on logs
-var stackTraceRegex = regexp.MustCompile(`\.go:\d+\s\+0x`)
```

Comment:
I moved this to the local scope.

Reply:
Hm so now the regex will be compiled multiple times instead and not just once.

Reply:
Wow! Wow! Wow! This is true. Thanks! I'll revert this change.